Frogies-Project/Assets/Scripts/Core/GlobalSceneManager.cs

```diff
@@ -73,7 +72,7 @@ public class GlobalSceneManager : MonoBehaviour
 
         private bool _isPaused = false;
 
-        private HashSet<BasicEntity> _entities;
+        public HashSet<BasicEntity> Entities { get; private set; }
```

Comment:
Would be great to design separate components for this. But I guess it is out of score of the ticket